[18.0][MIG] account_payment_order_vendor_email: Migration to 18.0

xaviedoanhduy opened the pull request:
a standard migration

Murtaza-OSI committed:
account_payment_order_vendor_email

oca-travis committed:
Update account_payment_order_vendor_email.pot

OCA-git-bot committed:
README.rst

pedrobaeza committed:
account_payment_order_vendor_email: Adapt module to native payment refactoring

OCA-git-bot committed:
account_payment_order_vendor_email 14.0.2.0.0

Nikul-OSI committed:
[MIG] account_payment_order_vendor_email: Migrated to v16

Commit:
[UPD] Update account_payment_order_vendor_email.pot

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (11 of 11 strings)

Translation: bank-payment-16.0/bank-payment-16.0-account_payment_order_vendor_email
Translate-URL: https://translation.odoo-community.org/projects/bank-payment-16-0/bank-payment-16-0-account_payment_order_vendor_email/it/

xaviedoanhduy committed:
[IMP] account_payment_order_vendor_email: pre-commit auto fixes

xaviedoanhduy committed:
[MIG] account_payment_order_vendor_email: Migration to 18.0

nguyenminhchien commented on account_payment_order_vendor_email/models/account_payment_mode.py:
```diff
@@ -24,77 +24,76 @@ class PaymentOrder(models.Model):
+                    template.with_context(**context).send_mail(rec.id, force_send=True)
+                    rec.message_post(
+                        body=self.env._(
+                            "An email is not able to send to %s vendor.", partner_name
```
the message is wrong, the email will be sent in this case.

xaviedoanhduy replied:
hello @nguyenminhchien. 
Thank you for catching this! I’ve prepared a [PR](https://github.com/OCA/bank-payment/pull/1459) to resolve it.

nguyenminhchien replied:
great! thanks. you could update it directly in your migration commit (in this PR).

xaviedoanhduy replied:
okay, I added it. Can you update your review?

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (213 of 213 strings)

Translation: bank-payment-18.0/bank-payment-18.0-account_payment_order
Translate-URL: https://translation.odoo-community.org/projects/bank-payment-18-0/bank-payment-18-0-account_payment_order/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (101 of 101 strings)

Translation: bank-payment-18.0/bank-payment-18.0-account_banking_mandate
Translate-URL: https://translation.odoo-community.org/projects/bank-payment-18-0/bank-payment-18-0-account_banking_mandate/it/

xaviedoanhduy committed:
[FIX] account_payment_order_vendor_email: change message content when template is sent successfully
